openmetadata-ui/src/main/resources/ui/src/utils/CustomizableLandingPageUtils.tsx

```diff
@@ -55,13 +55,24 @@ export const getAddWidgetHandler =
     if (
       placeholderWidgetKey === LandingPageWidgetKeys.EMPTY_WIDGET_PLACEHOLDER
     ) {
+      const lowestWidgetLayout = currentLayout.reduce(
```

Comment:
I feel we need to capture the position on emptyWidget and pass it here. This logic will again add widget at bottom no?